### ISSUE TITLE
Result page is missing a main heading

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4084]
## Area or Component
Result page

## Quick Summary
There's no H1 element on the results page. 

## Accessibility Violation
Page should contain a level-one heading: https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=AxeChrome

## Action Requested
Add a level-one heading to the page (we can probably use "Results" in the navigation bar for this - that's the equivalent experience for sighted users)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=16:16
